### ISSUE TITLE
custom faction editor:

### ISSUE THREAD

**Issue by FlankerDude** [dcs-liberation/dcs_liberation#789]
Yes you can make your own factions with a text editor, but its very tedious and i think that many people would enjoy this feature. So it would be awesome to just have a little editor window were you can choose your aircraft, ground, ships etc. units for your faction with their according liveries.

**Comment by clutchtomcat** [comment]
Not only tedious, but I am constantly having issues with syntax or missing punctuation because I am not a coder who knows how these things work. The often used response of "then learn to code" is borderline gatekeeping for those of us with full-time lives who just need stick time in an immersive campaign.

**Comment by Mustang-25** [comment]
I don't think this is a feature that will be addressed any time soon (but you never know). In the mean time, for anyone that does want to try their hand at building custom factions, [the wiki page](https://github.com/dcs-liberation/dcs_liberation/wiki/Custom-Factions) has been updated with more info to try and make this as easy as possible.
